dashboard/modules/serve/serve_agent.py

```diff
@@ -45,7 +45,7 @@ async def get_version(self, req: Request) -> Response:
             status=aiohttp.web.HTTPOk.status_code,
```

Comment:
why don't we have the status get API?

Reply:
Per-design discussion we removed it, we will only have a general metadata get endpoint. The `serve status` CLI can pull the status information from this endpoint.